lms/static/sass/features/_course-experience.scss

```diff
@@ -412,13 +412,24 @@
       }
     }
   }
+
+  .fa-chevron-right {
+    transform: rotateY(0deg) #{"/*rtl: rotateY(180deg)*/"};
+    font-size: inherit;
+
+    &.fa-rotate-90 {
+      transform: rotate(90deg) #{"/*rtl: rotate(90deg)*/"};
+      font-size: inherit;
+    }
+  }
 }
 
 #expand-collapse-outline-all-button {
   float: right;
   background-color: transparent;
   color: $black;
   cursor: pointer;
+  font-weight: 600;
```

Comment:
I don't think this is needed. On edx.org it's not currently semi-bold: 
![image](https://user-images.githubusercontent.com/1615421/83299923-14558680-a1c5-11ea-9f7e-971bfd782cd0.png)